dashboard/src/components/AppView/AccessURLTable/AccessURLItem/AccessURLIngressHelper.test.ts

```diff
@@ -38,6 +38,27 @@ describe("GetURLItemFromIngress", () => {
       tlsHosts: ["foo.bar"],
       expectedURLs: ["https://foo.bar", "http://not-foo.bar"],
     },
+    {
+      description: "it should ignore an ingress if it has no hosts",
```

Comment:
Does this still show an entry for the Ingress in the AccessURLTable, though without any clickable links? I think we'd still want to show that an Ingress exists, but that the hosts are not correctly defined.